editor/src/viewport_tools/tool_message_handler.rs

```diff
@@ -103,6 +103,20 @@ impl MessageHandler<ToolMessage, (&DocumentMessageHandler, &InputPreprocessorMes
 
 				update_working_colors(&self.tool_state.document_tool_data, responses);
 			}
+			SelectRandomPrimaryColor => {
+				// Select a random primary color (rgba) based on an UUID
+				let document_data = &mut self.tool_state.document_tool_data;
+
+				let random_number = generate_uuid();
+				let r = (random_number >> 24) as u8;
+				let g = (random_number >> 16) as u8;
+				let b = (random_number >> 8) as u8;
+				let a = random_number as u8;
```

Comment:
This looks nice and clean! The one thing is that I think it would be more useful if it's always opaque, so can you please modify this so the alpha value is always 1? Thanks!

Reply:
I just pushed the commit to change that. Thanks for the feedback!